src/test/resources/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolderAlias.java

```diff
@@ -0,0 +1,56 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.SuppressWarningsHolder
+aliasList = com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum
```

Comment:
It is not same as `<property name="aliasList" value="ParameterNumber=paramnum" />`
As you reference in issue.
You should have `aliasList=ParameterNumber=paramnum`
Or second = to be escaped.

Reply:
Or we need both formats in tests

Reply:
Or we need both formats in tests

Reply:
@romani the issue https://github.com/checkstyle/checkstyle/issues/11641 is all about supporting the simple name `aliasList=ParameterNumber=paramnum`.

 Right now it only supports the fully qualified name https://github.com/checkstyle/checkstyle/pull/11542#issuecomment-1126821468  

I will most probably send pr for fixing this by today https://github.com/checkstyle/checkstyle/issues/11641.

Reply:
Ok.